### PR TITLE
Readme patch 1

### PR DESCRIPTION
**Pull request by CPQE**

### CODE REVIEW

**Comment by CPQE** [comment]
merging comment on readme

**Comment by cyrusParvereshi** [comment]
oops

File: setup.py [hunk 1]
```diff
@@ -16,8 +16,9 @@
     author='Ben Johnson',
     author_email='bkitej@gmail.com',
     packages=find_packages(),
+    python_requires=">=3.8",
     install_requires=[
-        'pickle5',
+        'jupyter',
```
Comment (bkitej) [inline setup.py:21]: Removed. It runs without Jupyter though.

I've been getting frustrated with the interface when trying to clear the cache, might write some magics just to make it a bit easier to work with

File: setup.py [hunk 2]
```diff
@@ -16,8 +16,9 @@
     author='Ben Johnson',
     author_email='bkitej@gmail.com',
     packages=find_packages(),
+    python_requires=">=3.8",
```
Comment (bkitej) [inline setup.py:19]: This thing is so dumb I bet it could run on Python 1

**Pull request closed without merging**